Source/WebKit/Shared/AuxiliaryProcess.cpp

```diff
@@ -259,10 +263,8 @@ void AuxiliaryProcess::applyProcessCreationParameters(AuxiliaryProcessCreationPa
     SecureCoding::applyProcessCreationParameters(WTFMove(parameters));
 #endif
 #if ENABLE(CORE_IPC_SIGNPOSTS)
-    if (parameters.shouldEnableIPCSignposts)
-        IPC::Connection::forceEnableSignposts();
-    if (parameters.shouldEnableStreamingIPCSignposts)
-        IPC::StreamClientConnection::forceEnableSignposts();
+    // Note: some messages have already been sent at this point.
+    IPC::setSignpostsEnabled(parameters.shouldEnableIPCSignposts);
```

Comment:
I made two separate parameters because it seemed like the overhead of tracing StreamClientConnection is higher (way more of those messages than standard IPC::Connection)--would probably be nice to keep those as separate toggles.